department-of-veterans-affairs/abd-vro
# MCP-2132: Pass correct evidence to PDF

## What was the problem?

Referring to the diagram on the roadmap (https://github.com/department-of-veterans-affairs/abd-vro/wiki/VRO-v2-Roadmap) this PR essentially links the box called "Upload Summary Doc" to the rest of the routes, thus completing the picture.

The details of the the actual PDF upload, as well as the rest of the BIP API still need to be worked out.

Associated tickets or Slack threads:

- [MCP-2132](https://amida.atlassian.net/browse/MCP-2132)

## How does this fix it?[^1]

Add "fetch PDF" and "Upload PDF" to the camel routes.

## How to test this PR
- TBD


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
